app/components/CheckURLBanner/index.tsx

```diff
@@ -7,7 +7,16 @@ interface Props {
   onHide: () => void;
 }
 
+export const skipCheckURLBanner = () => {
+  if (typeof window === "undefined") return true;
+  return window.localStorage.getItem("checkURLBanner") === "skip";
+};
+
 export const CheckURLBanner: FC<Props> = ({ onHide }) => {
+  const onDontRemind = () => {
+    window.localStorage.setItem("checkURLBanner", "skip");
+    onHide();
```

Comment:
👍  As mentioned in the first pull request, a cookie would make more sense to not annoy the people. Localstorage works too.